packages/list-view/tests/list_view_test.js

```diff
@@ -56,13 +58,15 @@ test("should render an empty view when there is no content", function() {
         style: 'height:' + emptyViewHeight + 'px;'
       });
 
-  view = Ember.ListView.create({
-    content: content,
-    height: height,
-    rowHeight: rowHeight,
-    itemViewClass: itemViewClass,
-    emptyView: emptyView
-  });
+  Ember.run(function(){
+    view = Ember.ListView.create({
```

Comment:
Unfortunately this doesn't appear related to your PR, rather just something we will need to fix sometime afterwards https://github.com/emberjs/list-view/issues/196